lib/board.rb

```diff
@@ -21,13 +22,15 @@ def print_visual
     #{visual["A"][3]} | #{visual["B"][3]} | #{visual["C"][3]} | #{visual["D"][3]} | #{visual["E"][3]} | #{visual["F"][3]} | #{visual["G"][3]}
     #{visual["A"][2]} | #{visual["B"][2]} | #{visual["C"][2]} | #{visual["D"][2]} | #{visual["E"][2]} | #{visual["F"][2]} | #{visual["G"][2]}
     #{visual["A"][1]} | #{visual["B"][1]} | #{visual["C"][1]} | #{visual["D"][1]} | #{visual["E"][1]} | #{visual["F"][1]} | #{visual["G"][1]}
-    #{visual["A"][0]} | #{visual["B"][0]} | #{visual["C"][0]} | #{visual["D"][0]} | #{visual["E"][0]} | #{visual["F"][0]} | #{visual["G"][0]}"
+    #{visual["A"][0]} | #{visual["B"][0]} | #{visual["C"][0]} | #{visual["D"][0]} | #{visual["E"][0]} | #{visual["F"][0]} | #{visual["G"][0]}
+    "
```

Comment:
Smart to add another line to provide some breaks in the visuals in the terminal.